model/const.go

```diff
@@ -5,6 +5,7 @@ const (
 	EventPull   = "pull_request"
 	EventTag    = "tag"
 	EventDeploy = "deployment"
+	EventBranch = "branch"
```

Comment:
I think I overlooked one thing. Introducing a new Event type is actually quite a bit of work and is used in a lot of different places, including for security purposes to determine when secrets are injected.

Can you provide more information about the branch event type? Could we use the push event type for Gogs instead? Just want to understand if we __need__ a new event type